custom_dashboard/grafana_dashboards/home-dashboard.json

```diff
@@ -415,7 +415,7 @@
           },
           "editorMode": "builder",
           "exemplar": false,
-          "expr": "Expensive_Services_Detail{job=\"Most_Expensive_Services\", region=\"$region\", account=\"$Account\"}",
```

Comment:
It looks like you identified this to be a problem. Can you explain why was it so?
![image](https://github.com/XgridInc/xc3/assets/122358742/62d0604d-4275-4070-8a40-591a84ff831a)

Reply:
In **cost_metrics_of_expensive_services** Lambda function we are pushing metrics with job = Acount_id. This is why we are not able to get proper data with query we were using before. 

![image](https://github.com/XgridInc/xc3/assets/145985682/333b74a2-336a-4447-b926-33f257ae0205)